core/shared/src/main/scala/zio/metrics/MetricListener.scala

```diff
@@ -22,7 +22,7 @@ import zio.stacktracer.TracingImplicits.disableAutoTrace
  * A `MetricListener` is capable of taking some action in response to a metric
  * being recorded, such as sending that metric to a third party service.
  */
-trait MetricListener { self =>
+private[zio] trait MetricListener { self =>
```

Comment:
Not sure about this becoming private to zio. I am thinking about use cases where we would like to install a custom listener to bridge into an existing infrastructure.